Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -4,19 +4,53 @@
 
 namespace BootCamp.Chapter
 {
+    // ToDo: Change code so it only allows letters from an alphabet.
+
     /// <summary>
     /// Caesar cipher which supports all ASCII character (256 in total)
     /// </summary>
     public static class CaesarCipher
     {
+       
+        private const int MOD26 = 26;
```

Comment:
Not a clear name.

Reply:
Already changed constant  name to 'NumOfKeys'.. Thanks